server/test/lib/scene/scene.checkTrigger.test.js

```diff
@@ -7,15 +7,18 @@ const SceneManager = require('../../../lib/scene');
 const StateManager = require('../../../lib/state');
 
 const event = new EventEmitter();
+const house = {
+  get: fake.resolves([]),
+};
 
 describe('scene.checkTrigger', () => {
   it('should execute scene', async () => {
     const stateManager = new StateManager();
     const device = {
       setValue: fake.resolves(null),
     };
-    const sceneManager = new SceneManager(stateManager, event, device);
-    sceneManager.addScene({
+    const sceneManager = new SceneManager(stateManager, event, device, {}, {}, house, {});
+    await sceneManager.addScene({
```

Comment:
why await? addScene is not an async function

Reply:
oops, fixed :)